cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/worker/BasicWorkerContext.java

```diff
@@ -143,18 +147,22 @@ public Metrics getMetrics() {
   }
 
   public LoggingContext getLoggingContext() {
-    //TODO: Add adapter name if present later
     return new WorkerLoggingContext(program.getNamespaceId(), program.getApplicationId(), program.getId().getId(),
-                                    getRunId().getId(), String.valueOf(getInstanceId()), null);
+                                    getRunId().getId(), String.valueOf(getInstanceId()), adapterName);
```

Comment:
I looked around didn't find anything. Did you found something when you added to BasicMapReduceContext ?